airbyte-webapp/src/pages/SpeakeasyRedirectPage/SpeakeasyRedirectPage.tsx

```diff
@@ -0,0 +1,67 @@
+import { Suspense } from "react";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import { Navigate, useNavigate } from "react-router-dom";
+
+import { LoadingPage } from "components/LoadingPage";
+
+import { useAppMonitoringService } from "hooks/services/AppMonitoringService";
+import { useSpeakeasyRedirect } from "packages/cloud/services/cloudApi/useSpeakeasyRedirect";
+import { RoutePaths } from "pages/routePaths";
+import { ErrorOccurredView } from "views/common/ErrorOccurredView";
+
+export const SpeakeasyRedirectPage = () => {
+  const { trackError } = useAppMonitoringService();
+
+  return (
+    <SpeakeasyErrorBoundary trackError={trackError}>
+      <SpeakeasyLoginRedirect />
+    </SpeakeasyErrorBoundary>
+  );
+};
+
+const SpeakeasyLoginRedirect = () => {
+  const { redirectUrl } = useSpeakeasyRedirect();
+
+  return (
+    <Suspense fallback={<LoadingPage />}>
+      {redirectUrl ? <Navigate to={redirectUrl} /> : <CloudApiErrorView />}
+    </Suspense>
```

Comment:
This has not been tested end to end yet - I have only confirmed that `<CloudApiErrorView>` is displayed. @bgroff will do some e2e testing to make sure the redirect works as expected once the backend is in place.

Reply:
we thing this is appending the redirectUrl to some base path instead of navigating directly to the external redirectUrl